setup.cfg

```diff
@@ -15,6 +15,9 @@ classifier =
           Programming Language :: Python :: 3.4
           License :: OSI Approved :: BSD License
 
+[pycodestyle]
+max-line-length = 100
```

Comment:
I do like to keep to 80. /me shurgs.

Reply:
Bah... who only has 80 lines nowadays? :-)

Reply:
That's not the point :-) .

Reply:
I'm happy to set it back to 80, but often in python it feels like a trade-off between line length, keeping within pep8 multiline style, and adequately named members/functions. I feel like there isn't a nice way to style function declarations across multiple lines, so I chose to bump this instead. :D